src/components/page-layout.tsx

```diff
@@ -2,17 +2,25 @@ import * as React from "react";
 import Site from "../types/Site";
 import Header from "./header";
 import Footer from "./footer";
+import {HelperTool} from "./helper-tool"
 
 type Props = {
   _site: Site;
   children?: React.ReactNode;
+  document: any;
 };
 
-const PageLayout = ({ _site, children }: Props) => {
+const inProduction = !import.meta.env.DEV;
```

Comment:
Is this about local dev vs deployed, or do you really care about production (aka you do want this to show in sbx)?